.github/workflows/ci.yml

```diff
@@ -1,16 +1,19 @@
 name: Continuous Integration
 
 on:
-  - push
-  - pull_request
+  push: null
+  pull_request:
+    branches:
+      - master
 
 jobs:
-  tests:
+  integration:
```

Comment:
Why `integration`?

Reply:
> name: Continuous Integration

These are integration tests.
In an ideal world phpunit and php-cs-fixer are in separate jobs.
Please see https://github.com/ergebnis/php-library-template/blob/main/.github/workflows/integrate.yaml#L14

Reply:
They're unit tests, not integration tests. :-)

Reply:
Yes, if we move `php-cs-fixer` to another jobs.
May I do that?

Reply:
Sure.

Reply:
Done.

Reply:
:+1: